src/main/java/com/r307/arbitrader/service/TradingService.java

```diff
@@ -179,13 +183,13 @@ private void enterPosition(Spread spread) {
         LOGGER.debug("Short scale: {}", shortScale);
         LOGGER.debug("Long ticker ASK: {}", spread.getLongTicker().getAsk());
         LOGGER.debug("Short ticker BID: {}", spread.getShortTicker().getBid());
-        LOGGER.debug("Long fee percent: {}", longFeePercent);
-        LOGGER.debug("Short fee percent: {}", shortFeePercent);
+        LOGGER.debug("Trade fee percent: {}", tradeFee);
+        LOGGER.debug("Margin fee percent: {}", marginFee);
```

Comment:
See other comment. 
tradeFee is the long exchange trade fee, and marginFee is the sum of the short exchange tradeFee and marginFee. 
They should be logged as such:
LOGGER.debug("Long trade fee percent: {}", longTradeFee);
LOGGER.debug("Short trade and margin fee percent: {} + {} = {}", shortTradeFee, shortMarginFee, shortTotalFee);